Dockerfile

```diff
@@ -11,5 +11,8 @@ RUN cp $(find /opt/conda/envs/nf-core-proteomicslfq-*/share/msgf_plus-*/MSGFPlus
 # Add conda installation dir to PATH (instead of doing 'conda activate')
 ENV PATH /opt/conda/envs/nf-core-proteomicslfq-1.0dev/bin:$PATH
 
+# Add jar.............
+RUN cp $(find /opt/conda/envs/nf-core-proteomicslfq-*/share/msgf_plus-*/MSGFPlus.jar -maxdepth 0) $(find /opt/conda/envs/nf-core-proteomicslfq-*/bin/ -maxdepth 0)
```

Comment:
Just out of interest. Why is this required?

https://anaconda.org/bioconda/msgf_plus

Reply:
Yes it's unfortunate, but Java-based tools in conda that are called via a ".jar" file,
usually have to be wrapped in a shell script that calls "java -jar /.../share/actual.jar".
But we do this in our CPP wrapper already, so we just want the jar.
And since hardcoding the path to the jar would not be very future proof and needs to be
changed whenever we update versions, I chose a version with wildcards that copies the jar to a findable location in the PATH.
The alternative of dynamically modifying PATH or adding another env variable to find the jar later is just impossible with docker. I hate it for this. You can give it a try if you want.